src/controllers/cart.controller.ts

```
@@ -0,0 +1,488 @@
+        });
+    } catch (error) {
+        console.error('Error in getCart:', error);
+        res.status(500).json({
+            status: 'error',
+            message: 'Failed to fetch cart'
+        });
+    }
+};
+
+// Add item to cart
+export const addCartItem = async (req: Request<{}, {}, ICartItemCreate>, res: Response): Promise<void> => {
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)